### DISCUSSION TITLE
[Esercitazione 6 Dicembre 2023] Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#340] [Esercizi - Viaggi/Piazza] [answered]
**Esercizio 2**. Si consideri il sistema non-omogeneo di 4 equazioni in 5 incognite (ottenuto dal sistema omogeneo dell'esercizio #119 )

$$\begin{cases}x_1+x_2+2x_3+x_5=1\\
x_1-x_2+x_4+x_5=2\\
x_1-x_3-x_5=1\\
x_1+2x_2+5x_3+3x_5=1\end{cases}$$

- Applicare il metodo di Gauss e determinare un sistema a scala, $S\underline x=\underline c$ , equivalente al sistema dato.
- Verificare che il sistema a scala $S\underline x=\underline c$ è compatibile. (Otteniamo quindi la compatibilità del sistema iniziale)
- Sia $\Sigma$ l'insieme delle soluzioni del sistema iniziale. Scrivere $\Sigma$ nella forma:
$$\Sigma=Span(\underline w_1,...,\underline w_l)+\underline v_0$$

  per un opportuno $l\in \mathbb{N}$ e per opportuni vettori $\underline w_1,...,\underline w_l,\underline v_0\in \mathbb{R}^5$, con $\\{\underline w_1,...,\underline w_l\\}$ linearmente indipendenti; verificate in questo modo che vale il teorema di struttura.

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/dc84006e-ea95-4b79-8f27-1e4fdee0b3ea)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/8475bf94-89f7-41f6-a136-680ac7aa3b76)

**Comment by daromacarlo** [comment]
![Immagine 2023-12-07 191721](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/6e6fe922-f35c-4fec-903a-8d9686d63b6a)
![Immagine 2023-12-07 191735](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/4fd585e3-0020-457c-8c94-51832b2dee9e)
![Immagine 2023-12-07 191747](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/1f74c16a-0782-48ac-b2e0-b593fccd93d2)
![Immagine 2023-12-07 192008](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/fefc9305-ad1f-492f-a0ad-9f339c0470ff)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/9f565891-840a-4526-854b-8eb5ead8818a)
